### ISSUE TITLE
Remember "Use C++" choice

### ISSUE THREAD

**Issue by MyzTyn** [apertium/apertium-viewer#3]
I successfully created a new language and language pair through the bootstrap script. The language pair linked the new language and English (apertium-eng). I added some words to the morphological and bilingual dictionary. I tried to use apertium-viewer to view the translation and got errors about lttoolbox (upgrade or something) and OutOfMemory. I did allocate more memory to the Java application but still the same errors. Can anyone help me? Also, I can provide more details about errors. 

**Development:** VMware Ubuntu 20.04LTS
**How did I install:** Apertium core using packaging

![image](https://user-images.githubusercontent.com/69483519/168382244-2b610e7c-5664-4ac3-8735-02cc24ed9fde.png)

**Comment by unhammer** [comment]
ping  @nordfalk

**Comment by marcriera** [comment]
I can recreate this if I select "Java" instead of "C++ version" at the top left with `apertium-eng-cat` and `apertium-spa-cat`.

**Comment by MyzTyn** [comment]
Hmm. Interesting. If I select "Java" or "C++ version", I still get same errors with `apertium-eng` and custom language.

**Comment by MyzTyn** [comment]
@unhammer I don't think it was apterium-viewer's issue. Because, I just did tested the apterium's command line to translate it. Then same error message show that `out of memory`. And the apterium didn't use whole 8 GB memory.

![Screenshot 2022-05-13 201625](https://user-images.githubusercontent.com/69483519/168403388-a11687ea-021a-4ffe-a69a-4d8331cfc53e.png)

**Comment by marcriera** [comment]
> Hmm. Interesting. If I select "Java" or "C++ version", I still get same errors with `apertium-eng` and custom language.

I've realized it's probably due to this: https://github.com/apertium/apertium-init/issues/31. Open the `modes.xml` from your new pair and change `apertium-tagger -g $2` to `apertium-tagger -gx $2` for the direction English > custom language. Run `make` again and test if it works under "C++ version". The other direction (custom language > English) should be working under "C++ version" without editing anything.

**Comment by MyzTyn** [comment]
It worked! But is there a way for me to set using `C++ version` by default every time I open the application?
![image](https://user-images.githubusercontent.com/69483519/168427396-5d4950d5-e8ab-4cff-89d9-1b57dbc026fc.png)

**Comment by unhammer** [comment]
OK, so one issue was the English tagger -x thing, already tracked in init#31, another that lttoolbox.java hasn't caught up with the recent changes C++ so that even freshly init-ed language pairs don't work with viewer – so it'd be nice if apertium-viewer at least remembered the "Use C++" choice. 

@nordfalk perhaps "Use C++" should be default for new users?

**Comment by nordfalk** [comment]
I agree - "use C++" choice should be the default. 

If you release a new version before I get around it then let me know - I need to change a text file in my server (javabog.dk) that indicates newest release, to inform users that they can upgrade
